src/app/credExplorer/basicPagerank.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import stringify from "json-stable-stringify";
+import * as tf from "@tensorflow/tfjs-core";
+
+import {Graph} from "../../core/graph";
+
+/**
+ * Run eigenvector centrality, and print results to console.
+ */
+export default function basicPagerank(graph: Graph<any, any>): void {
+  tf.tidy(() => {
+    const {nodes, addressToIndex, markovChain} = graphToMarkovChain(graph);
+    const stationaryDistribution = findStationaryDistribution(markovChain);
```

Comment:
is there a reason not to have `const stationaryDistribution = findstationaryDistribution(markovChain).dataSync()`?

Reply:
I don't see any substantive reason one way or the other, but I find the way that I wrote it more readable (and (side-benefit) it takes fewer lines of code).